models/userDao.js

```diff
@@ -0,0 +1,37 @@
+const dataSource = require('./dataSource')
+
+const getUserByEmail = async (email) => {
+	const result = await dataSource.query(`
+		SELECT 
+			id,
+			name,
+			email,
+			password,
+			birthday,
+      phone_number
+		FROM users
+		WHERE email=?`, [email]
+	);
+
+	return result[0];
+}
```

Comment:
가독성을 위해 해당 부분의 indentation과 tabsize를 수정해주세요!

```suggestion
const getUserByEmail = async (email) => {
  const result = await dataSource.query(`
    SELECT 
      id,
      name,
      email,
      password,
      birthday,
      phone_number
    FROM users
    WHERE email=?`, [email]
    );
  return result[0];
}
```